tools/monitoring/monitor-parachain-staking.ts

```diff
@@ -0,0 +1,95 @@
+import { typesBundle } from "moonbeam-types-bundle";
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import * as fs from "fs";
+import { xxhashAsU8a } from "@polkadot/util-crypto";
+import { u8aConcat, u8aToHex } from "@polkadot/util";
+
+const main = async () => {
+  const wsProvider = new WsProvider("ws://localhost:56992");
+  const polkadotApi = await ApiPromise.create({
+    provider: wsProvider,
+    typesBundle: typesBundle as any,
+  });
+
+  const collatorAddress = "0x37006dd9d226425c901c5d7f9434c2fb8ac3f533";
+  const parachainCollatorStorageKey = u8aToHex(
+    u8aConcat(
+      xxhashAsU8a("ParachainStaking", 128),
+      xxhashAsU8a("CollatorState2", 128),
+      xxhashAsU8a(collatorAddress, 64),
+      collatorAddress
+    )
+  );
+
+  // const startBlock = 1;
+  const startBlock = 401000;
```

Comment:
It would  be good if some of this hardcoded values were parametric

Reply:
Yes, it was for specific purposes, I'll make it with params